pages/linux/needrestart.md

```diff
@@ -0,0 +1,36 @@
+# needrestart
+
+> Checks which daemons need to be restarted after library upgrades.
+> More information: <https://github.com/liske/needrestart>.
+
+- List outdated processes:
+
+`needrestart`
+
+- Interactively restart services:
+
+`sudo needrestart`
+
+- Run in [v]erbose or [q]uiet mode:
+
+`needrestart -{{v|q}}`
```

Comment:
Since we don't really describe flags alone, but only what they do in combination with the command, what do you think abaout
```suggestion
- List outdated processes in [v]erbose or [q]uiet mode:

`needrestart -{{v|q}}`
```
If listing processes doesn't really fit here, how about
```suggestion
- Interactively restart services in [v]erbose or [q]uiet mode:

`sudo needrestart -{{v|q}}`
```